native/src/utils/calendarRangeUtils.ts

```diff
@@ -0,0 +1,49 @@
+type MarkedDateType = {
+  selected: boolean
+  startingDay?: boolean
+  endingDay?: boolean
+}
+
+export const getMarkedDates = (startDate: string, endDate: string): Record<string, MarkedDateType> => {
+  const markedDateStyling = {
+    color: '#FBDA16',
+    textColor: '#000',
+  }
```

Comment:
Hardcoding the style here like this prevents adjusting to different themes, e.g. for Malte or a future dark mode. Instead, you could for example pass it as a parameter.